### DISCUSSION TITLE
Есть пара вопросов.

### DISCUSSION THREAD

**Discussion by fbatyrbayev** [malikzh/NCANode#74] [Q&A] [answered]
Привет. Хотел заюзать ваш продукт, есть пара вопросов.
HttpInteractor и RabbitMqInteractor классах вы принудительно вызываете  System.gc(), это необходимость вызванная каким то багом?

**Comment by malikzh** [comment, marked as answer]
Привет. Помню была проблема с тем, что память всю съедало. Вот этот костыль
вроде бы решил проблему)

пт, 15 янв. 2021 г., 0:41 fbatyrbayev <notifications@github.com>:

**Reply by fbatyrbayev** [reply]
А режим с rabbitmq кто нить юзает, мне кажется просто он "бажный" на первый взгляд. Взять хотя бы, что очередь убивается автоделитом, а дальше уже не принимает запросы?

**Reply by malikzh** [reply]
Вообще я планировал сделать RabbitMqInteractor - deprecated и потом удалить его. Практически все подключается через HTTP и мало кто пользуется этим функционалом.

**Reply by developeractiv** [reply]
Здравствуйте. Я хотел использовать Ваш проект в связке с RabbitMQ, т.к. у нас микросервисная архитектура, нам проще масштабировать (кол-во нод) и вопрос с балансировкой решен.  В планах еще прикрутить Redis к ноде, чтобы файлы передавать не через шину, а через кэш. Предложение не удалять RabbitMqInteractor.

**Reply by malikzh** [reply]
Здравствуйте. Я на этой неделе планирую делать релиз версии 3.0. 
Я пока поддержку AMQP не реализовывал, но на Spring'е это не проблема, имхо.
